warehouse/models/staging/payments/stg_payments__elavon_transactions.sql

```diff
@@ -0,0 +1,56 @@
+WITH source AS (
+    SELECT * FROM {{ source('elavon_external_tables', 'transactions') }}
+    WHERE dt IS NOT NULL
+    AND execution_ts IS NOT NULL
```

Comment:
nit: I don't think these can be null, it'd probably be more robust to test them instead of filter